templates/_candlepin_logging.conf.epp

```diff
@@ -0,0 +1,10 @@
+<%- | Hash[String[1], Candlepin::LogLevel] $loggers | -%>
+<%- $loggers.each |$logger, $log_level| { -%>
+
+log4j.logger.<%= $logger %>=<%= $log_level %>
+<%- } -%>
+<%- unless 'org.candlepin' in $loggers { -%>
+
+# uncomment to enable debug logging in candlepin.log:
+#log4j.logger.org.candlepin=DEBUG
```

Comment:
Do we want to keep this comment here that a user can't do anything with? Or would lead them astray?

Reply:
My reasoning was that it can help users who quickly want to debug, but I wasn't quite sure.

Reply:
I'd lean to drop to prevent the inevitable bug reports about this gets wiped out every time they run the installer and then having to explain they ought to use the installer itself.